examples/inverse/mne_cov_power.py

```diff
@@ -34,14 +34,17 @@
 raw_fname = meg_path / 'sample_audvis_filt-0-40_raw.fif'
 raw = mne.io.read_raw_fif(raw_fname)
 
+
+
+
 # %%
 # Compute empty-room covariance
 # -----------------------------
 # First we compute an empty-room covariance, which captures noise from the
 # sensors and environment.
 
-raw_empty_room_fname = op.join(
-    data_path, 'MEG', 'sample', 'ernoise_raw.fif')
+raw_empty_room_fname = (
+    data_path/ 'MEG'/ 'sample'/ 'ernoise_raw.fif')
```

Comment:
Style:

operators (like `/` here) should have one space on either side. Cf.

- https://mne.tools/dev/install/contributing.html#adhere-to-standard-python-style-guidelines
- https://peps.python.org/pep-0008/#other-recommendations

Reply:
yeah. I corrected it